web/app/controllers/Application.java

```diff
@@ -46,33 +49,48 @@
      *             routing is managed client side
      * @return {Result} build output index.html resource
      */
-    public static Result serveAsset(String path) {
+    private static Result serveAsset(String path) {
```

Comment:
It's not for some reason that I haven't figured out yet, taking it out from here and the signatures in `conf/routes` breaks asset routing from Ember's node server, which I use for local dev and  proxy to :9000. It 404's then,  so I'm leaving it in, but will revisit when I can investigate the play route a bit more.
@mars-lan 
```
...
GET /assets/assets/fonts/Source-Sans-Pro-600/Source-Sans-Pro-600.woff 404 8.185 ms - 1613
...
```